utils/hare-bootstrap

```diff
@@ -281,7 +286,12 @@ fi
 abort_if_RC_leader_election_is_impossible
 
 read _ join_ip <<< $(get_server_nodes | grep -w $(node-name))
-read _ join_peers <<< $(get_server_nodes | grep -vw $(node-name))
+
+# get_server_nodes will produce multiple lines. We'll join them with space.
+# Note: in case when get_server_nodes is empty list, consequent awk and tr will exit with non-zero code.
```

Comment:
Isn't this line too long? Please, make sure it fits 80 chars width.

Reply:
Addressed